alphacsc/datasets/camcan.py

```diff
@@ -0,0 +1,202 @@
+""" Cambridge Centre for Ageing and Neuroscience (CamCAN) dataset
+Data available at: https://www.cam-can.org/index.php?content=dataset
+
+Shafto, M.A., Tyler, L.K., Dixon, M., Taylor, J.R., Rowe, J.B., Cusack, R.,
+Calder, A.J., Marslen-Wilson, W.D., Duncan, J., Dalgleish, T., Henson, R.N.,
+Brayne, C., CamCAN, & Matthews, F.E. (2014). The Cambridge Centre for Ageing
+and Neuroscience (CamCAN) study protocol: a cross-sectional, lifespan,
+multidisciplinary examination of healthy cognitive ageing. BMC Neurology,
+14(204). doi:10.1186/s12883-014-0204-1
+https://bmcneurol.biomedcentral.com/articles/10.1186/s12883-014-0204-1 
+"""
+import os
+from os.path import join
+from copy import deepcopy
+
+import mne
+import numpy as np
+import pandas as pd
+from joblib import Memory
+
+from mne_bids import BIDSPath, read_raw_bids
+
+try:
+    from ..utils.signal import split_signal
+except ValueError:
+    from alphacsc.utils.signal import split_signal
+
+
+DATA_DIR = "/storage/store/data/"
```

Comment:
alphacsc is meant to be used by anyone not just us at Inria

Reply:
I tried to mimic and reproduced how paths were handled in `alphacsc/datasets/hcp.py`